[2018-09-05T14:41:25.119] [ERROR] errorLog - TypeError: ctx.render is not a function
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:11:19)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at clientRoute (G:\node\node-service-frame\server\routers\clientRouter.js:49:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at bodyParser (G:\node\node-service-frame\node_modules\koa-bodyparser\index.js:86:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2018-09-05T14:41:25.122] [ERROR] errorLog - TypeError: ctx.render is not a function
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:11:19)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at clientRoute (G:\node\node-service-frame\server\routers\clientRouter.js:49:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at bodyParser (G:\node\node-service-frame\node_modules\koa-bodyparser\index.js:86:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
[2018-09-05T14:41:25.125] [ERROR] errorLog - TypeError: ctx.render is not a function
    at _callee$ (G:/node/node-service-frame/server/routers/clientRouter.js:11:19)
    at tryCatch (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:62:40)
    at Generator.invoke [as _invoke] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:296:22)
    at Generator.prototype.(anonymous function) [as next] (G:\node\node-service-frame\node_modules\regenerator-runtime\runtime.js:114:21)
    at step (G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:17:30)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:35:14
    at new Promise (<anonymous>)
    at new F (G:\node\node-service-frame\node_modules\core-js\library\modules\_export.js:36:28)
    at G:\node\node-service-frame\node_modules\babel-runtime\helpers\asyncToGenerator.js:14:12
    at clientRoute (G:\node\node-service-frame\server\routers\clientRouter.js:49:21)
    at dispatch (G:\node\node-service-frame\node_modules\koa-compose\index.js:42:32)
    at bodyParser (G:\node\node-service-frame\node_modules\koa-bodyparser\index.js:86:11)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
